MVVM Module/Tests/Feed UI/Helpers/FeedViewController+TestHelpers.swift

```diff
@@ -10,6 +10,10 @@ extension FeedViewController {
 		refreshControl?.simulatePullToRefresh()
 	}
 
+	func simulateTapOnError() {
+		feedErrorView?.button.simulate(event: .touchUpInside)
+	}
+
```

Comment:
Following my previous comment on `errorView` it is better to access `errorView` here instead of casting the `tableHeaderView` to `ErrorView` on the test side.

```suggestion
	func simulateTapOnError() {
		errorView?.button.simulate(event: .touchUpInside)
	}
```